routes/movies.ts

```diff
@@ -0,0 +1,34 @@
+import express, { NextFunction, Request, Response } from "express"
+import { validate } from "../util";
+import { isAuthApiKey } from "../auth/auth";
+import { Movies } from "../controllers/movies-class";
+import { validationResult } from "express-validator";
+
+export const moviesRouter = express.Router();
+
+moviesRouter.get('/data', isAuthApiKey, async (req: Request, res: Response, next: NextFunction) => {
+    return res.send(await new Movies().getMoviesData());
+});
```

Comment:
It is more readable like this
```suggestion
moviesRouter.get(
    '/data',
    isAuthApiKey,
    async (req: Request, res: Response, next: NextFunction) => {
        return res.send(await new Movies().getMoviesData());
    }
);
```